src/css/main.less

```diff
@@ -925,6 +920,27 @@ fieldset[disabled] .btn-info.active {
     main {
       flex: 1;
       padding-left: @container-fluid-padding;
+
+      .contextual-bar {
+        align-items: center;
+        display: flex;
+        padding: 16px 0;
+
+        .app-list-controls {
+          flex: 1;
+        }
+
+        .breadcrumbs {
+          color: #C3C9D7;
+          flex: 1;
+
+          h2 {
+            margin: 0;
+            padding: 0;
+            font-size: 18px;
```

Comment:
Alternatively we could be more expressive:

`@h1-font-size, @h2-font-size` etc.